gulpfile.js

```diff
@@ -81,6 +83,22 @@ var DEFINES = {
   PDFJS_NEXT: false,
 };
 
+function safeSpawnSync(command, parameters, options) {
+  // Windows has problems executing commands outside a shell.
+  if (process.platform === 'win32') {
```

Comment:
It seems to me that it's mainly an issue when `spawn`ing a `gulp` task, but otherwise I think it's fine without it.